core/secret.py.example

```diff
@@ -0,0 +1,2 @@
+# create access token from tradier.com
+API_Access_Token="<Access-Token>"
```

Comment:
How do you obtain API access token?

Reply:
i am working on it. changing the api again

Reply:
Cool, which one are you planning to use now?

Reply:
the previous one was not deprecated, it was transferred. Using it again